src/App.jsx

```diff
@@ -1,10 +1,76 @@
-import React from 'react';
+import React, { Component } from "react";
+import axios from "axios";
 
-function App() {
-  return (
-    <div>
-    </div>
-  );
+class App extends Component {
+  state = {
+    query: "",
+    results: [],
+    message: ""
+  };
+
+  onSubmitFormHandler = async e => {
+    e.preventDefault();
+    let response = await axios.post("https://cocktails-api-team1.herokuapp.com/api/v1/cocktails", {
+      params: {
+        query: "name"
+      }
+    });
+    if (response.data.status === 400) {
+      debugger;
```

Comment:
a `debugger`, really?

Reply:
We removed all debuggers right as you commented about them. Sorry about that.